Dockerfile

```
@@ -1,4 +1,4 @@
-FROM node:13.8.0-alpine3.11
+FROM node:12.16.1-alpine3.11
```

I'd recommend using:

```suggestion
FROM node:12-alpine
```